packages/terser/src/constants.ts

```diff
@@ -1,2 +1,3 @@
 export const taskInfo = Symbol('taskInfo');
 export const freeWorker = Symbol('freeWorker');
+export const workerPoolWorkerFlag = 'WorkerPoolWorker';
```

Comment:
is there a reason we cannot also use a symbol here?

Reply:
This is sent as `workerData` in the `Worker` constructor, so it must be clonable.

References:
- https://nodejs.org/dist/latest-v18.x/docs/api/worker_threads.html#workerworkerdata
- https://developer.mozilla.org/en-US/docs/Web/API/Web_Workers_API/Structured_clone_algorithm